README.md

```diff
@@ -113,9 +152,9 @@ with AudioStream(processor=feature_processor) as stream:
 
 Matchmaker currently supports the following alignment methods:
 
-- `"dixon"`: On-line time warping algorithm by S. Dixon (2005).
-- `"arzt"`: On-line time warping algorithm adapted from Brazier and Widmer (2020) (based on the work by Arzt et al. (2010))
-- `"hmm"`: Hidden Markov Model-based score follower by Cancino-Chacón et al. (2023), based on the state-space score followers by Duan et al. (2011) and Jiang and Raphael (2020).
+- `"dixon"`: On-line time warping algorithm by S. Dixon (2005). Supports audio input only.
```

Comment:
I think both OLTW could support MIDI as well, but this requires a little bit more work (which we can address in the next release), maybe we can add "MIDI support will come in the next release"?

Reply:
https://github.com/pymatchmaker/matchmaker/pull/13/commits/7ac5682d9676603a376419e633587e7386907c71 👍